ios-target-app/Home/Models/TargetResponse.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  TargetResponse.swift
+//  ios-target-app
+//
+//  Created by Raul Piñeres Carrera on 1/11/22.
+//
+
+import Foundation
+
+// MARK: - Target Response
+struct TargetResponse: Codable {
+  let target: TargetRaw
+  let matchConversation: MatchConversation?
+  let matchedUser: MatchedUser?
+  
+  enum CodingKeys: String, CodingKey {
+    case target
+    case matchConversation = "match_conversation"
+    case matchedUser = "matched_user"
+  }
+}
+
+// MARK: Target Raw
+struct TargetRaw: Codable {
```

Comment:
1. Can this be just `Target`?
2. Please use only `Decodable` if not intended to encode this

```suggestion
struct Target: Codable {
```